lib/src/runner/vm/platform.dart

```diff
@@ -21,29 +30,64 @@ class VMPlatform extends PlatformPlugin {
 
   VMPlatform();
 
-  StreamChannel loadChannel(String path, TestPlatform platform) {
+  StreamChannel loadChannel(String path, TestPlatform platform) =>
+      throw new UnimplementedError();
+
+  Future<RunnerSuite> load(String path, TestPlatform platform,
+      SuiteConfiguration suiteConfig) async {
     assert(platform == TestPlatform.vm);
 
-    var isolate;
-    var channel = StreamChannelCompleter.fromFuture(() async {
-      var receivePort = new ReceivePort();
+    var receivePort = new ReceivePort();
+    Isolate isolate;
+    try {
+      isolate = await _spawnIsolate(path, receivePort.sendPort);
+    } catch (error) {
+      receivePort.close();
```

Comment:
just a question: does this need to be closed in case of success (at some point later in the function)? or does IsolateChannel/something else handle that.